### DISCUSSION TITLE
More Documentation

### DISCUSSION THREAD

**Discussion by Linuxury** [ChrisTitusTech/linutil#874] [General]
## Future Prof New entries 
This tool is in desperate needs for better documentation for not just the name of new entries but also for what they are intended to be used. Not to forget the lack of documentation on keyboard mapping not added anywhere 😞 

## Describe the solution you'd like
A good solution would be to integrate a simple template to fill that information at the time of submission or to be able to modify them if they have already been added.

## Describe alternatives you've considered
A would suggest something like this to start with

[Category]
[Sub category] if any
[Name]
[Description] what it does is a must for people new to Linux, since this is a tool that aims to help new user it should not be ignored in my opinion.
[Hot keys] if any, for example how to pull Rofi or how to use neovim, etc... you guys get my point.

## Additional context
I know it's great if you know those things by heart, but won't kill you if you help the new guy at the same time.

**Comment by koibtw** [comment]
I'm moving this to a discussion since there's not much going on here.
